.github/copilot-instructions.md

```
@@ -0,0 +1,51 @@
+    using simple yet powerful YAML configuration files. It generates C++ firmware that can be compiled and flashed to
+    these devices, allowing users to control them remotely through home automation systems.
+ 
+    This repo is the source for the primary documentation for users of ESPHome, published on [esphome.io](https://esphome.io).
+*   **Business Domain:** Internet of Things (IoT), Home Automation.
+
+## Core Technologies & Stack
+
+*   **Languages:** HTML, CSS, Markdown, Go templates
+*   **Frameworks & Runtimes:** Hugo and Pagefind
+*   **Key Libraries/Dependencies:**
+    *   **Hugo:** For building the static site.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)